packages/wdio-cli/src/utils.ts

```diff
@@ -531,42 +533,39 @@ export async function getAnswers(yes: boolean): Promise<Questionnair> {
 
     const projectProps = await getProjectProps(process.cwd())
     const isProjectExisting = Boolean(projectProps)
+    const nameInPackageJsonIsNotCreateWdioDefault = projectProps?.packageJson?.name !== 'my-new-project'
     const projectName = projectProps?.packageJson?.name ? ` named "${projectProps.packageJson.name}"` : ''
-    const questions = [
-        /**
-         * in case the `wdio config` was called using a global installed @wdio/cli package
-         */
-        ...(!isProjectExisting
-            ? [{
-                type: 'confirm',
-                name: 'createPackageJSON',
-                default: true,
-                message: `Couldn't find a package.json in "${process.cwd()}" or any of the parent directories, do you want to create one?`,
-            }]
-            /**
-             * in case create-wdio was used which creates a package.json with name "my-new-project"
-             * we don't need to ask this question
-             */
-            : projectProps?.packageJson?.name !== 'my-new-project'
-                ? [{
-                    type: 'confirm',
-                    name: 'projectRootCorrect',
-                    default: true,
-                    message: `A project${projectName} was detected at "${projectProps?.path}", correct?`,
-                }, {
-                    type: 'input',
-                    name: 'projectRoot',
-                    message: 'What is the project root for your test project?',
-                    default: projectProps?.path,
-                    // only ask if there are more than 1 runner to pick from
-                    when: /* istanbul ignore next */ (answers: Questionnair) => !answers.projectRootCorrect
-                }]
-                : []
-        ),
-        ...QUESTIONNAIRE
+
+    const projectRootQuestions = [
+        {
+            type: 'confirm',
+            name: 'createPackageJSON',
+            default: true,
+            message: `Couldn't find a package.json in "${process.cwd()}" or any of the parent directories, do you want to create one?`,
+            when: !isProjectExisting
+        },
+        {
+            type: 'confirm',
+            name: 'projectRootCorrect',
+            default: true,
+            message: `A project${projectName} was detected at "${projectProps?.path}", correct?`,
+            when: isProjectExisting && nameInPackageJsonIsNotCreateWdioDefault
```

Comment:
Can we move `isProjectExisting && nameInPackageJsonIsNotCreateWdioDefault` into a const somewhere above and use it for both of the `when` callbacks?

Reply:
@christian-bromann yes, I can do it. Seems like it won't affect any behavior for now (while `package.json` doesn't get modified somewhere in the middle of running `wdio config`)

Reply:
The CI has failed after the change. Unit tests are saying the following about `getProjectProps` in `constants.ts`:

> TypeError: getProjectProps is not a function"

`console.log(getProjectProps)` shows `undefined`. I find it difficult to understand why is that 🤔

Reply:
Note that unit tests may mock out entire packages or even native functions like `fetch`.